components/ai_chat/browser/ai_chat_tab_helper.cc

```diff
@@ -458,6 +489,15 @@ std::string AIChatTabHelper::BuildLlama2Prompt(std::string user_message) {
   // Always use a generic system message
   std::string system_message =
       l10n_util::GetStringUTF8(IDS_AI_CHAT_LLAMA2_SYSTEM_MESSAGE_GENERIC);
+  base::Time::Exploded exploded;
+  base::Time::Now().LocalExplode(&exploded);
+  std::string date_string = base::StringPrintf(
+      "%02d:%02d %s, %s %02d, %d", exploded.hour, exploded.minute,
+      GetDayOfWeekString(exploded.day_of_week).c_str(),
+      GetMonthString(exploded.month - 1).c_str(), exploded.day_of_month,
+      exploded.year);
+  std::string today_system_message =
+      base::ReplaceStringPlaceholders(system_message, {date_string}, nullptr);
```

Comment:
Should we use AM/PM? Based on https://chromium.googlesource.com/chromium/src/+/lkgr/base/time/time.h#30, you might look at using `base/i18n/time_formatting.h`

Reply:
oh this looks great, actually tried looking for this but without success 🙏